Tests/ValTests/Utils/AnnotatedValFileTest.swift

```diff
@@ -103,6 +105,26 @@ extension XCTestCase {
     }
   }
 
+  /// Returns the Val test cases in the suite at `path` relative to `swiftFile` or `nil` if no
+  /// such suite could be located.
+  ///
+  /// The suite is searched at `path` relative to `swiftFile`. If no such directory exists, `path`
+  /// is searched in the root of the resource bundle associated with the current Swift module.
```

Comment:
```suggestion
  /// The suite is sought at `path` relative to `swiftFile`. If no such directory exists, `path`
  /// is sought in the root of the resource bundle associated with the current Swift module.
```
(one searches/searched a haystack for a needle; one seeks/sought the needle in the haystack)

But even with my suggestion it's not very clear what's intended here.  What does it mean to seek a path?

Reply:
This one is definitely a French/English issue! Makes perfect sense to "chercher un chemin".

I want to say that the method is seeking the test suite at two different locations: `path` relative to `swiftFile` and, if it's not there, `path` relative to the bundle's resource URL. Perhaps the confusion is due to the fact that test suite and directory are used interchangeably here.